packages/monaco/src/browser/style/index.css

```diff
@@ -259,3 +259,11 @@
 .monaco-editor .peekview-widget .monaco-list-row.focused.selected .codicon {
   color: var(--theia-list-activeSelectionForeground) !important;
 }
+
+.quick-input-titlebar {
+  background-color: var(--theia-input-background);
```

Comment:
It might be odd to use the input background here, but it seems vscode sets this styling dynamically (inline) and seem to use the same color as the input field itself. We can refactor to use a variable that uses the same color if needed.

Reply:
Sounds good!